src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -12,8 +12,27 @@
  */
 public class FieldXY {
 
-    public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+    public void moveRobot(Robot robot, int coordinateX, int coordinateY) {
+        robot.getCoordinateX();
+        robot.getCoordinateY();
+        robot.getDirection();
+        int[] toX = new int[coordinateX - robot.getCoordinateX()];
+        int[] toY = new int[coordinateY - robot.getCoordinateY()];
```

Comment:
@EliosPride you can get here exeption if result is negative

Reply:
@al-pavlenko correct mistake, willn't have any exception